src/utils.ts

```diff
@@ -258,24 +259,20 @@ export async function askToConfigureEnvironment(
           message, DialogResponses.yes, DialogResponses.no);
 
   if (result === DialogResponses.yes) {
-    telemetryContext.properties.errorMessage =
-        `${operation} operation failed and user configures project`;
-
-    let res: boolean;
-    res =
-        await ProjectEnvironmentConfiger.configureProjectEnvironmentAsPlatform(
-            context, channel, telemetryContext, platform, deviceRootPath,
-            scaffoldType);
-    if (res) {
-      const message =
-          `Configuration of project environmnet done. You can run the ${
-              operation.toLocaleLowerCase()} operation now.`;
-      channelShowAndAppendLine(channel, message);
-      vscode.window.showInformationMessage(message);
-    }
+    telemetryContext.properties.errorMessage = `${
+        operation} operation failed and user configures project environment.`;
+
+    await ProjectEnvironmentConfiger.configureProjectEnvironmentAsPlatform(
```

Comment:
>ProjectEnvironmentConfiger.configureProjectEnvironmentAsPlatform [](start = 10, length = 64)

As we discussed before, I think we don't need to ask user to config the  Arduino project iif it is not compatible with current Arduino project template. We can config it silently for user and remove the 'Arduino' project type from "ConfigProjectEnvironment" command.

Reply:
This function is a utility function and will handle configuration for both container project and arduino project.